spec/services/idv/steps/ipp/verify_step_spec.rb

```diff
@@ -22,7 +23,6 @@
   describe '#call' do
     it 'works' do
       result = step.call
-      expect(result).to be_nil
```

Comment:
Removing this assertion because the call method doesn't return nil anymore (it returns a url that the user gets redirected to). The new return value is ignored though so it's not worth enforcing it in a spec that might later confuse someone

I think the point of this spec right now is just making sure no errors are thrown so this seems fine to me